File: java/Napakalaki/src/NapakalakiGame/DeathBadConsequence.java

package NapakalakiGame;

/**
 *
 * @author Sofía Almeida Bruno
 * @author María Victoria Granados Pozo
 */
public class DeathBadConsequence extends NumericBadConsequence {
    /**
     * 
     * Constructor
     * @param t Descripción del mal rollo
     */
    DeathBadConsequence(String t) {
        super(t, Player.MAXLEVEL, NumericBadConsequence.MAXTREASURES, NumericBadConsequence.MAXTREASURES);
    }
    
    /**
     * 
     * @return String con información del mal rollo
     */
    @Override
    public String toString() {
        return super.toString();
    }
}

File: ruby/Napakalaki/lib/death_bad_consequence.rb
#encoding: utf-8

require_relative "bad_consequence"

#Sofía Almeida Bruno
#María Victoria Granados Pozo

module NapakalakiGame
  class DeathBadConsequence < NumericBadConsequence   
    def initialize (t)
      super(t, Player.getMaxLevel, BadConsequence.getMaxTreasures, BadConsequence.getMaxTreasures)
      @death = true
    end
    
    def to_s
      "\n\tTexto: #{getText} \n\tMuerto"
    end
  end
end
